_data/authors.yaml

```diff
@@ -440,3 +440,10 @@ fgregorio:
   emailhash: "9aaa99ccaa78472b0b3988ef5620f475"
   job_title: "Architect"
   bio: "Architect at Sofis Solutions."
+hcummins:
+  name: "Holly Cummins"
+  email: "hcummins@redhat.com"
+  emailhash: "44cbcba2dc44254f2ef292469b896363"
+  job_title: "Software Engineer"
+  twitter: "gsmet_"
+  bio: "Software Engineer, working at Red Hat on Quarkus."
```

Comment:
```suggestion
  bio: "Software Engineer, working at Red Hat on Quarkus."
  
```

Reply:
Thanks, @machi1990, good spots!  I can't see what the difference is in the bio line (but I know there must be one).

Reply:
the suggestion was adding a new line at the end of the file :-)

Reply:
Ohhhh, that would explain why I couldn't see it in the GH UI. :)